lib/client.js

```diff
@@ -863,6 +863,10 @@ class Parser {
       resume(client)
     }
   }
+
+  finish () {
+    llhttp.exports.llhttp_finish(this.ptr)
+  }
```

Comment:
Can you put this before `destroy()`? I've tried to put all the callback methods at the end of the class and all regular methods before that.